app/frontend/wallet/backend-types.ts

```diff
@@ -105,8 +86,15 @@ export type PoolRecommendationResponse = StakePoolInfoExtended & {
   isInRecommendedPoolSet: boolean
 }
 
+export declare type Token = {
+  policyId: string
+  assetName: string
+  quantity: string // possibly huge amounts
```

Comment:
we should introduce bigint for that - native is probably a bit too soon as Safari switched only recently, so I'd use some library for that